### PR TITLE
Bug 1859591: version controller: honor system CA bundle when checking route health

### PR DESCRIPTION
**Pull request by stlaz**
/assign @marun

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -336,13 +338,19 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 		controllerContext.EventRecorder,
 	)
 
+	systemCABundle, err := ioutil.ReadFile("/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem")
+	if err != nil {
+		// this may fail route-health checks in proxy environments
+		klog.Warningf("Unable to read system CA from /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem: %v", err)
```
Comment (mfojtik) [inline pkg/operator/starter.go:344]: why not make this check during the route healthcheck and report it in message to user?

Comment (stlaz) [reply pkg/operator/starter.go:344]: it's not a check, you're loading the full file which might be quite long, you don't want this io operation appearing all the time

Comment (mfojtik) [reply pkg/operator/starter.go:344]: i think a simple "file exists" check (os.Stat) would be cheap?

Comment (stlaz) [reply pkg/operator/starter.go:344]: heh, I see why this is confusing, I added a new argument but did not wire it :facepalm: 

The contents of the file are being used. I do not care whether it exists or not.

**Pull request merged**